### ISSUE TITLE
Lighting.Technology is gone and toolbox constantly hangs (V16)

### ISSUE THREAD

**Issue by UnsatisfiedPrius** [Roblox/future-is-bright#159]
I just opened up FIB V16 and the technology option in Lighting mysteriously disappeared.
![image](https://user-images.githubusercontent.com/42261094/56192059-3311b180-5ff3-11e9-96e7-8df8d12cf1ec.png)

Not only that, the toolbox constantly hangs from any sort of interaction from clicking on models to just typing in the search bar.

Is there any sort of hint on when this will be fixed, or even another FIB release? I build exclusively with Future Is Bright's detailed shaders and none of my builds would look right without it.

**Comment by vvv331** [comment]
I got the same thing there, Lighting.Technology gone.
![image](https://user-images.githubusercontent.com/49684723/56213456-ba4b3d80-6064-11e9-9a98-c4ef9d9dd503.png)

**Comment by UnsatisfiedPrius** [comment]
Are you having the toolbox issues too?

**Comment by Edo-UK** [comment]
![image](https://user-images.githubusercontent.com/49719893/56287583-a9f29b80-6114-11e9-883f-7d8568e1467f.png)
I'm having the same issue

**Comment by Edo-UK** [comment]
FIB also freezes when I load up a game, and it takes a couple of minutes for it to stop freezing.

**Comment by SethizFTW** [comment]
V15 doesn't use lighting to enable FIB you must go in settings to do so.

**Comment by LoganDark** [comment]
Even the command bar can't access Lighting.Technology.

**Comment by SheriffTaco** [comment]
Did anyone ever find out a workaround to this or if it's going to be fixed? I'm unable to use it right now.

**Comment by LoganDark** [comment]
There is no workaround. It's a server-side issue with ROBLOX.

**Comment by SheriffTaco** [comment]
Do you think it will be fixed soon?

**Comment by LoganDark** [comment]
V17.

**Comment by UnsatisfiedPrius** [comment]
Wish we knew when V17 was coming.

**Comment by LoganDark** [comment]
Betas usually come quarterly or so. A beta release would usually show up at around this point.

**Comment by reece07** [comment]
Any updates?

**Comment by LoganDark** [comment]
@reece07 Nope. Like I said, wait for V17.

**Comment by Advalys** [comment]
If anyone is having a problem with the lighting and wants to use Future Lighting but can't, I have a fresh start file with the Lighting Technology put to future, so if anyone wants to start making something but can't use the nice lighting, here's a starter file for it.
[Future Lighting Preset.zip](https://github.com/Roblox/future-is-bright/files/3117042/Future.Lighting.Preset.zip)

**Comment by LoganDark** [comment]
@Advalys If you want, I could make a tool that takes an RBXLX file and sets its Lighting.Technology to Future.

It won't be an exe though, it'll probably be a Python thing.

**Comment by Advalys** [comment]
@LoganDark I don't really have a problem with it since I only use like 3 places with future lighting and I don't really need to make any new ones, and if I did I could just make a copy of one of the places and just clear it. Thanks for the offer though. :)

**Comment by vvv331** [comment]
I know how to set FiB lighting, but I dont tell you how
Hint: Use .rbxlx file and change it.

**Comment by reece07** [comment]
@vvv331 someone said how to earlier in the comment chain lol

**Comment by UnsatisfiedPrius** [comment]
@vvv331 What is even the point of trying to keep it a secret anyway?

**Comment by lightingtech** [comment]
same problem here.

**Comment by SheriffTaco** [comment]
Does anyone know if there is a way to manually change your place's technology to FIB? If not, is there a link somewhere to V15?

**Comment by vvv331** [comment]
@SheriffTaco Yes, it's possible by changing a .rbxlx file, try finding "Technology" and set it to 42 (I don't really know).
And yes, this is how to turn on FiB if you can't.

**Comment by elyI7** [comment]
@vvv331 Where do you find that .rbxlx file?

**Comment by Firopense** [comment]
I guess you need to search for that in the folder itself
